entities/generic-entities.ent

```diff
@@ -619,3 +620,7 @@ use &deng;! -->
 
 <!-- SLE HA, old productname entity -->
 <!ENTITY hasi                   "&ha;">
+
+<!-- ALP, old entities -->
+<!ENTITY alpshort               "ALP">
```

Comment:
As ALP was renamed to SUSE Linux Framework One, maybe we should get rid of alpshort completely? :thinking: 

Of course, this is out of scope for this PR, but as there is no official abbreviation, maybe better not offer it?
If some oldish guides still need them, they can define it in their entity list.

Reply:
i would go even further and cancel all `&alp-*` related entities:
```
&alp;                 # to be replaced by &slfo;
&alpshort;            # to be cancelled
&alp-installer;       # to be replaced by &agama;
```